### PR TITLE
(DOCSP-17232) iOS integration title fixes

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-17232

### Staged Changes (Requires MongoDB Corp SSO)

- [iOS SDK > Build for tvOS](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-17232-iOS-integration-title-fixes/sdk/ios/integrations/tvos/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/ios.txt [hunk 1]
```diff
@@ -193,11 +193,11 @@ Realm Apps
...
 - :doc:`Link User Identities </sdk/ios/advanced-guides/link-user-identities>`
 
-Integrations
-------------
+Integration Guides
+------------------
 
-- :doc:`SwiftUI & Combine </sdk/ios/integrations/swiftui>`
-- :doc:`tvOS </sdk/ios/integrations/tvos>`
+- :doc:`Use Realm Database with SwiftUI and Combine </sdk/ios/integrations/swiftui>`
```
Comment (mongomoe) [inline source/sdk/ios.txt:199]:
```suggestion
- :doc:`SwiftUI & Combine </sdk/ios/integrations/swiftui>`
```

File: source/sdk/ios/integrations.txt [hunk 2]
```diff
@@ -5,8 +5,8 @@ Integrations - iOS SDK
 .. toctree::
    :titlesonly:
 
-   SwiftUI & Combine </sdk/ios/integrations/swiftui>   
-   tvOS </sdk/ios/integrations/tvos>
+   Use Realm Database with SwiftUI and Combine </sdk/ios/integrations/swiftui>
```
Comment (mongomoe) [inline source/sdk/ios/integrations.txt:8]:
```suggestion
   SwiftUI & Combine </sdk/ios/integrations/swiftui>
```

File: source/sdk/ios/integrations.txt [hunk 3]
```diff
@@ -5,8 +5,8 @@ Integrations - iOS SDK
...
    :titlesonly:
 
-   SwiftUI & Combine </sdk/ios/integrations/swiftui>   
-   tvOS </sdk/ios/integrations/tvos>
+   SwiftUI & Combine </sdk/ios/integrations/swiftui>
+   Build for tvOS </sdk/ios/integrations/tvos>
 
-- :doc:`SwiftUI & Combine </sdk/ios/integrations/swiftui>`
-- :doc:`tvOS </sdk/ios/integrations/tvos>`
+- :doc:`Use Realm Database with SwiftUI and Combine </sdk/ios/integrations/swiftui>`
```
Comment (mongomoe) [inline source/sdk/ios/integrations.txt:11]:
```suggestion
- :doc:`SwiftUI & Combine </sdk/ios/integrations/swiftui>`
```

**Review by dacharyc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**